lib/plugins/acts_as_customizable/lib/acts_as_customizable.rb

```diff
@@ -155,7 +155,7 @@ def respond_to?(method, include_private = false)
         def for_custom_field_accessor(method_symbol)
           match = /\Acustom_field_(?<id>\d+)=?\z/.match(method_symbol.to_s)
           if match
-            custom_field = CustomField.find_by_id(match[:id])
+            custom_field = available_custom_fields.find { |cf| cf.id.to_s == match[:id] }
```

Comment:
@NobodysNightmare the only option that comes to mind is continuing the pattern that is used up until here meaning providing a default implementation on acts_as_customizable for getting a single custom field defined on the customizable and then overriding it in work package. 

Having the method itself would be good as we would reduce the dependency from active_record methods. But it would mean having two implementations, which I wouldn't like.  

??? You see, I don't have a clear idea here.